config/config.go

```diff
@@ -46,8 +46,9 @@ type Config struct {
 	DisableGRPCReflection bool `envconfig:"DISABLE_GRPC_REFLECTION" default:"false"`
 	// Trace header, when this HTTP header is present CB will add the value to log/trace contexts
 	TraceHeaderName string `envconfig:"TRACE_HEADER_NAME" default:"x-trace-id"`
-	// When we match this HTTP header prefix, we forward append the values to grpc metadata
-	HTTPHeaderPrefix string `envconfig:"HTTP_HEADER_PREFIX" default:""`
```

Comment:
by removing this we are breaking backwards compatibility, we should preserve this field